pkg/kubelet/kubelet.go

```diff
@@ -2838,7 +2838,7 @@ func validateBandwidthIsReasonable(rsrc *resource.Quantity) error {
 }
 
 func extractBandwidthResources(pod *api.Pod) (ingress, egress *resource.Quantity, err error) {
-	str, found := pod.Annotations["kubernetes.io/ingress-bandwidth"]
+	str, found := pod.Annotations["net.beta.kubernetes.io/ingress-bandwidth"]
```

Comment:
fwiw, i just started evaluating this last week, i cannot imagine there is wide use as it didn't get much in the way of documentation to my knowledge